src/applications/vaos/appointment-list/components/cards/confirmed/VideoVisitProvider.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { getPractitionerDisplay } from '../../../../services/appointment';
+
+export default function VideoVisitProvider({ participants }) {
+  if (!participants) {
+    return null;
+  }
+
+  const practitionerDisplay = getPractitionerDisplay(participants);
+
+  return (
+    <dl className="vads-u-margin--0">
+      <dt className="vads-u-font-weight--bold">You'll be meeting with</dt>
```

Comment:
On va.gov, we try to use a right single quote instead of apostrophes (`You’ll` as opposed to `You'll`).  I don't know the original reasoning, but as a shortcut, you can use `Shift + Alt + ]` to type this right single quote, or just copy paste it from elsewhere.

Reply:
Ah, yes that apostrophe, got me again! LOL